src/previewCode.ts

```diff
@@ -4,9 +4,8 @@ import { ManimShell } from './manimShell';
 import { EventEmitter } from 'events';
 import { Logger } from './logger';
 
-const PREVIEW_COMMAND = `\x0C checkpoint_paste()\x1b`;
```

Comment:
just for clarification:
removing `\x1b` - won't break the https://github.com/bhoov/manim-notebook/issues/18#issuecomment-2431146809 ?

Reply:
thanks

Reply:
[related](https://github.com/Manim-Notebook/manim-notebook/issues/18#issuecomment-2453461077)